### PR TITLE
virtio_serial:fix large_file_transfer issue

### PR DESCRIPTION
**Pull request by nanliu-r**
ID: 1773428

Signed-off-by: Nana Liu <nanliu@redhat.com>

### CODE REVIEW

**Comment by nanliu-r** [comment]
Test results:
 (1/4) repeat1.Host_RHEL.m8.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.virtio_serial_file_transfer.tcp_socket.long_time_test.from_host_to_guest: PASS (671.54 s)
 (2/4) repeat1.Host_RHEL.m8.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.virtio_serial_file_transfer.tcp_socket.long_time_test.from_guest_to_host: PASS (674.07 s)

 (1/4) repeat1.Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.8.x86_64.io-github-autotest-qemu.virtio_serial_file_transfer.tcp_socket.long_time_test.from_host_to_guest: PASS (756.52 s)
 (2/4) repeat1.Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.8.x86_64.io-github-autotest-qemu.virtio_serial_file_transfer.tcp_socket.long_time_test.from_guest_to_host: PASS (731.60 s)

**Comment by nanliu-r** [comment]
@vivianQizhu Please help to review this, thanks a lot.

File: qemu/tests/virtio_serial_file_transfer.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ def check_output(sender, output):
 
 @error_context.context_aware
 def transfer_data(params, vm, host_file_name=None, guest_file_name=None,
-                  sender='both'):
+                  sender='both', host_clean=True):
```
Comment (vivianQizhu) [inline qemu/tests/virtio_serial_file_transfer.py]: Please add doc string for it, and I will prefer name it `clean_file`, to handle guest file together.

Comment (nanliu-r) [reply qemu/tests/virtio_serial_file_transfer.py]: Thanks and update.

File: qemu/tests/virtio_serial_file_transfer.py [hunk 2]
```diff
@@ -200,7 +201,8 @@ def transfer_data(params, vm, host_file_name=None, guest_file_name=None,
                   port_name, guest_file_name, guest_action))
     result = _transfer_data(session, host_cmd, guest_cmd, transfer_timeout, sender)
     clean_cmd = params['clean_cmd']
-    os.remove(host_file_name)
+    if clean_file:
+        os.remove(host_file_name)
     if os_type == "windows":
         guest_file_name = guest_file_name.replace("/", "\\")
     session.cmd('%s %s' % (clean_cmd, guest_file_name))
```
Comment (vivianQizhu) [inline qemu/tests/virtio_serial_file_transfer.py]: I think the above lines should be included in `if clean_file:` too.

**Pull request merged**